app/models/menu.rb

```diff
@@ -28,5 +28,9 @@ module FlexCommerce
   #
   class Menu < FlexCommerceApi::ApiBase
     has_many :menu_items
+    # This model has a path attribute so path can no longer be used to modify the path
+    def self.path(params = nil, record = nil)
+      super(params.except("path"), record)
```

Comment:
If params can be nil, would you not be best doing `Hash(params).except("path")` or `params ? params.except("path") : nil`